### PR TITLE
Ravendreth from WoWHead scraper

### PR DESCRIPTION
**Pull request by elidion**
Added the scraped version of ravendreth, still needs to be followed along with a video for proper guide formatting.

### CODE REVIEW

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 1]
```diff
@@ -1,11 +1,743 @@
...
+T The Greatest Duelist|QID|57381|N|Unknown.|
+
+A On The Road Again|QID|56942|M|65.2,66.2|Z|Revendreth|N|From Rendle.|LVL|57|PRE|56829|
+C On The Road Again|QID|56942|Z|Revendreth|N|Unknown.|
+C On The Road Again 1|QID|56942|Z|Revendreth|QO|1|N|Ride the Darkhaven Carriage.|
+T On The Road Again|QID|56942|M|69,81.8|Z|Revendreth|N|To Lord Chamberlain.|
+
+A Rebels on the Road|QID|56955|M|69,81.8|Z|Revendreth|N|From Lord Chamberlain.|LVL|57|PRE|56942|
+C Rebels on the Road|QID|56955|M|66.4,64.4|Z|Revendreth|N|Unknown.|
+C Rebels on the Road 1|QID|56955|M|66.4,64.4|Z|Revendreth|QO|1|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Stoneborn Rebel.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:55]: extra html codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 2]
```diff
@@ -1,11 +1,743 @@
...
+T The Authority of Revendreth|QID|57179|M|57.4,49.6|Z|Revendreth|N|To Sire Denathrius.|
+
+
+
+
+; The Accuser Storyline
+A I Don't Get My Hands Dirty|QID|57161|M|58,49.4|Z|Revendreth|N|From Lord Chamberlain.|LVL|57|
+C I Don't Get My Hands Dirty|QID|57161|M|61.8,49.4|Z|Revendreth|N|Unknown.|
+C I Don't Get My Hands Dirty 1|QID|57161|M|61.8,49.4|Z|Revendreth|QO|1|N|<span>Clear Penance Bridge.|
+C I Don't Get My Hands Dirty 2|QID|57161|M|58.8,48.4|Z|Revendreth|QO|2|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Kill Credit: Normal Monster.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:108]: extra html codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 3]
```diff
@@ -1,11 +1,743 @@
...
+C The Penitent Hunt|QID|57136|M|47.8,70|Z|Revendreth|N|Unknown.|
+C The Penitent Hunt 1|QID|57136|M|47.8,70|Z|Revendreth|QO|1|N|Speak to the Fearstalker.|
+C The Penitent Hunt 2|QID|57136|M|50.4,61.4|Z|Revendreth|QO|2|N|Crimelord Tiana hunted.|
+C The Penitent Hunt 3|QID|57136|Z|Revendreth|QO|3|N|Yoshai the Merciless hunted.|
+C The Penitent Hunt 4|QID|57136|Z|Revendreth|QO|4|N|Houndmaster Loksey hunted.|
+T The Penitent Hunt|QID|57136|N|Unknown.|
+
+A Devour This|QID|57164|M|47.8,70|Z|Revendreth|N|From The Fearstalker.|LVL|58|
+C Devour This|QID|57164|M|42.8,68.2|Z|Revendreth|N|Unknown.|
+C Devour This 1|QID|57164|M|42.8,68.2|Z|Revendreth|QO|1|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Ravenous Thirstlurker.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:233]: Extra HTML codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 4]
```diff
@@ -1,11 +1,743 @@
...
+T Chasing Madness|QID|57405|M|38,60.6|Z|Revendreth|N|To Theotar.|
+
+A My Terrible Morning|QID|57426|M|38,60.6|Z|Revendreth|N|From Theotar.|LVL|58|
+C My Terrible Morning|QID|57426|Z|Revendreth|N|Unknown.|
+C My Terrible Morning 1|QID|57426|Z|Revendreth|QO|1|N|Marquis Pummeler slain.|
+T My Terrible Morning|QID|57426|N|Unknown.|
+
+A Unbearable Light|QID|57427|M|38,60.6|Z|Revendreth|N|From Theotar.|LVL|58|
+C Unbearable Light|QID|57427|M|22.2,53.4|Z|Revendreth|N|Unknown.|
+C Unbearable Light 1|QID|57427|M|22.2,53.4|Z|Revendreth|QO|1|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Blistering Ash Ghoul.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:320]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 5]
```diff
@@ -1,11 +1,743 @@
...
+
+A Into the Light|QID|60566|M|32,47.6|Z|Revendreth|N|From Theotar.|LVL|58|
+C Into the Light|QID|60566|Z|Revendreth|N|Unknown.|
+C Into the Light 1|QID|60566|Z|Revendreth|QO|1|N|Ride the elevator to Sinfall.|
+T Into the Light|QID|60566|M|30.4,45.6|Z|Revendreth|N|To Breakfist.|
+
+A Securing Sinfall|QID|57724|M|30.4,45.6|Z|Revendreth|N|From Breakfist.|LVL|58|
+C Securing Sinfall|QID|57724|M|29,42.8|Z|Revendreth|N|Unknown.|
+C Securing Sinfall 1|QID|57724|M|29,42.8|Z|Revendreth|QO|1|N|<span>Sinfall cleared out.|
+C Securing Sinfall 2|QID|57724|M|28,41.4|Z|Revendreth|QO|2|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Lingering Monitor.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:364]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 6]
```diff
@@ -1,11 +1,743 @@
...
+
+A Comfortably Numb|QID|59716|M|60.2,69.4|Z|Revendreth|N|From Taskmaster Matyas.|LVL|57|
+C Comfortably Numb|QID|59716|Z|Revendreth|N|Unknown.|
+C Comfortably Numb 1|QID|59716|Z|Revendreth|QO|1|N|Matyas poisoned.|
+C Comfortably Numb 2|QID|59716|Z|Revendreth|QO|2|N|Nadjia's Sinstone.|
+T Comfortably Numb|QID|59716|M|60,69|Z|Revendreth|N|To Nadjia the Mistblade.|
+
+A The Field of Honor|QID|59724|M|60,69|Z|Revendreth|N|From Nadjia the Mistblade.|LVL|57|
+C The Field of Honor|QID|59724|M|60,79.4|Z|Revendreth|N|Unknown.|
+C The Field of Honor 1|QID|59724|M|60,79.4|Z|Revendreth|QO|1|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Bladewing Guard.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:486]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 7]
```diff
@@ -1,11 +1,743 @@
...
+A We're Gonna Need a Bigger Dredger|QID|57477|M|72.6,73.2|Z|Revendreth|N|From Rendle.|LVL|57|
+C We're Gonna Need a Bigger Dredger|QID|57477|Z|Revendreth|N|Unknown.|
+C We're Gonna Need a Bigger Dredger 1|QID|57477|Z|Revendreth|QO|1|N|[HIDDEN] Channeling Complete.|
+C We're Gonna Need a Bigger Dredger 2|QID|57477|Z|Revendreth|QO|2|N|Witness the ritual.|
+T We're Gonna Need a Bigger Dredger|QID|57477|M|72.8,73.8|Z|Revendreth|N|To Bootus.|
+
+A Running a Muck|QID|57481|M|72.8,73.8|Z|Revendreth|N|From Bootus.|LVL|57|
+C Running a Muck|QID|57481|M|72.8,73.8|Z|Revendreth|N|Unknown.|
+C Running a Muck 1|QID|57481|M|72.8,73.8|Z|Revendreth|QO|1|N|Bootus boarded.|
+C Running a Muck 2|QID|57481|M|75.4,77.2|Z|Revendreth|QO|2|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Rampaging Devourer.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:568]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 8]
```diff
@@ -1,11 +1,743 @@
...
+
+A Hunting an Inquisitor|QID|57929|M|73,52|Z|Revendreth|N|From Archivist Fane.|LVL|57|
+C Hunting an Inquisitor|QID|57929|M|76,51.8|Z|Revendreth|N|Unknown.|
+C Hunting an Inquisitor 1|QID|57929|M|76,51.8|Z|Revendreth|QO|1|N|Inquisitor Traian slain.|
+T Hunting an Inquisitor|QID|57929|M|73,52|Z|Revendreth|N|To Archivist Fane.|
+
+A Halls of Atonement: Your Absolution|QID|58092|M|73,52|Z|Revendreth|ELITE|N|[color=e6cc80]Dungeon: [/color]\nFrom Archivist Fane.|LVL|57|
+C Halls of Atonement: Your Absolution|QID|58092|M|14.2,48.7|Z|Halls of Atonement|N|Unknown.|
+C Halls of Atonement: Your Absolution 1|QID|58092|M|14.2,48.7|Z|Halls of Atonement|QO|1|N|Lord Chamberlain slain.|
+C Halls of Atonement: Your Absolution 2|QID|58092|Z|Halls of Atonement|QO|2|N|<span>Suggested players: 5.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:637]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 9]
```diff
@@ -1,11 +1,743 @@
...
+C Light Punishment 2|QID|57533|Z|Revendreth|QO|2|N|Mirror Fragment A.|
+C Light Punishment 3|QID|57533|Z|Revendreth|QO|3|N|Mirror Fragment B.|
+C Light Punishment 4|QID|57533|Z|Revendreth|QO|4|N|Mirror Fragment C.|
+C Light Punishment 5|QID|57533|Z|Revendreth|QO|5|N|Mirror Fragment D.|
+C Light Punishment 6|QID|57533|Z|Revendreth|QO|6|N|Mirror Fragment E.|
+T Light Punishment|QID|57533|M|24.2,49.4|Z|Revendreth|N|To Simone.|
+
+A When Only Ash Remains|QID|57534|M|24.2,49.4|Z|Revendreth|N|From Laurent.|LVL|58|
+C When Only Ash Remains|QID|57534|M|21.2,48|Z|Revendreth|N|Unknown.|
+C When Only Ash Remains 1|QID|57534|M|21.2,48|Z|Revendreth|QO|1|N|<table class="iconlist"> <tr><th><ul><li><var></var></li></ul></th><td>Ashen Ravager.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:684]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 10]
```diff
@@ -1,11 +1,743 @@
...
+A Escaping the Master|QID|57535|M|24.2,49.4|Z|Revendreth|N|From Laurent.|LVL|58|
+C Escaping the Master|QID|57535|Z|Revendreth|N|Unknown.|
+C Escaping the Master 1|QID|57535|Z|Revendreth|QO|1|N|Mirror Fragment.|
+T Escaping the Master|QID|57535|M|24.2,49.4|Z|Revendreth|N|To Laurent.|
+
+A Mirror Making, Not Breaking|QID|57536|M|24.2,49.4|Z|Revendreth|N|From Laurent.|LVL|58|
+C Mirror Making, Not Breaking|QID|57536|M|24.6,50.4|Z|Revendreth|N|Unknown.|
+C Mirror Making, Not Breaking 1|QID|57536|M|24.6,50.4|Z|Revendreth|QO|1|N|Follow Laurent and Simone.|
+C Mirror Making, Not Breaking 2|QID|57536|M|24.6,50.4|Z|Revendreth|QO|2|N|Speak to Laurent.|
+C Mirror Making, Not Breaking 3|QID|57536|M|24.2,49.4|Z|Revendreth|QO|3|N|<span>Laurent and Simone defended.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:697]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 11]
```diff
@@ -1,11 +1,743 @@
...
+
+A Moving Mirrors|QID|57571|M|24.2,49.4|Z|Revendreth|N|From Laurent.|LVL|58|
+C Moving Mirrors|QID|57571|M|21.8,50.4|Z|Revendreth|N|Unknown.|
+C Moving Mirrors 1|QID|57571|M|21.8,50.4|Z|Revendreth|QO|1|N|Simone freed.|
+C Moving Mirrors 2|QID|57571|Z|Revendreth|QO|2|N|Mirror Trap.|
+T Moving Mirrors|QID|57571|M|24.2,49.4|Z|Revendreth|N|To Simone.|
+
+A We Need More Power|QID|59427|M|24.2,49.4|Z|Revendreth|N|From Simone.|LVL|58|
+C We Need More Power|QID|59427|M|21.1,53.7|Z|Revendreth|N|Unknown.|
+C We Need More Power 1|QID|59427|M|21.1,53.7|Z|Revendreth|QO|1|N|<span>Anima collected.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:710]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 12]
```diff
@@ -1,11 +1,743 @@
...
+C Offer of Freedom 1|QID|59868|Z|Revendreth|QO|1|N|Compelled venthyr freed.|
+T Offer of Freedom|QID|59868|M|60.2,78.6|Z|Revendreth|N|To Nadjia the Mistblade.|
+
+
+
+
+; Sidequest Storyline -Tithes of Darkhaven
+A Bring Out Your Tithe|QID|60176|M|61.4,63.8|Z|Revendreth|N|From Mistress Mihaela.|LVL|57|
+C Bring Out Your Tithe|QID|60176|Z|Revendreth|N|Unknown.|
+C Bring Out Your Tithe 1|QID|60176|Z|Revendreth|QO|1|N|<span>Tithe anima collected.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:519]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 13]
```diff
@@ -1,11 +1,743 @@
...
+T An Uneventful Stroll|QID|57461|M|32,47.6|Z|Revendreth|N|To Theotar.|
+
+A Into the Light|QID|60566|M|32,47.6|Z|Revendreth|N|From Theotar.|LVL|58|
+C Into the Light|QID|60566|Z|Revendreth|N|Unknown.|
+C Into the Light 1|QID|60566|Z|Revendreth|QO|1|N|Ride the elevator to Sinfall.|
+T Into the Light|QID|60566|M|30.4,45.6|Z|Revendreth|N|To Breakfist.|
+
+A Securing Sinfall|QID|57724|M|30.4,45.6|Z|Revendreth|N|From Breakfist.|LVL|58|
+C Securing Sinfall|QID|57724|M|29,42.8|Z|Revendreth|N|Unknown.|
+C Securing Sinfall 1|QID|57724|M|29,42.8|Z|Revendreth|QO|1|N|<span>Sinfall cleared out.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:363]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 14]
```diff
@@ -1,11 +1,743 @@
...
+C The Accuser 3|QID|60506|QO|3|N|The Accuser hunted.|
+C The Accuser 4|QID|60506|QO|4|N|The Accuser confronted.|
+C The Accuser 5|QID|60506|QO|5|N|<span>Cast Door of Shadows to cross the barrier.|
+T The Accuser|QID|60506|M|51.6,59.2|Z|Revendreth|N|To The Accuser.|
+
+A A Reflection of Truth|QID|57159|M|51.6,59.2|Z|Revendreth|N|From The Accuser.|LVL|58|
+C A Reflection of Truth|QID|57159|Z|Revendreth|N|Follow the Accuser through the mirror and witness the truth of Revendreth.|
+C A Reflection of Truth 1|QID|57159|Z|Revendreth|QO|1|N|Mirror entered.|
+C A Reflection of Truth 2|QID|57159|Z|Revendreth|QO|2|N|Truth witnessed.|
+C A Reflection of Truth 3|QID|57159|Z|Revendreth|QO|3|N|<span>The Accuser protected.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:251]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 15]
```diff
@@ -1,11 +1,743 @@
...
+C Devour This 3|QID|57164|Z|Revendreth|QO|3|N|Disturbed Mite.|
+T Devour This|QID|57164|N|Unknown.|
+
+A The Accuser|QID|60506|N|Unknown.|LVL|58|
+C The Accuser|QID|60506|N|Unknown.|
+C The Accuser 1|QID|60506|QO|1|N|[HIDDEN] Mirror Impact.|
+C The Accuser 2|QID|60506|QO|2|N|[HIDDEN] Door of Shadows Received.|
+C The Accuser 3|QID|60506|QO|3|N|The Accuser hunted.|
+C The Accuser 4|QID|60506|QO|4|N|The Accuser confronted.|
+C The Accuser 5|QID|60506|QO|5|N|<span>Cast Door of Shadows to cross the barrier.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:244]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 16]
```diff
@@ -1,11 +1,743 @@
...
+C The Authority of Revendreth 6|QID|57179|Z|Revendreth|QO|6|N|Stand in the Sigil of the Harvester.|
+T The Authority of Revendreth|QID|57179|M|57.4,49.6|Z|Revendreth|N|To Sire Denathrius.|
+
+
+
+
+; The Accuser Storyline
+A I Don't Get My Hands Dirty|QID|57161|M|58,49.4|Z|Revendreth|N|From Lord Chamberlain.|LVL|57|
+C I Don't Get My Hands Dirty|QID|57161|M|61.8,49.4|Z|Revendreth|N|Unknown.|
+C I Don't Get My Hands Dirty 1|QID|57161|M|61.8,49.4|Z|Revendreth|QO|1|N|<span>Clear Penance Bridge.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:107]: Extra HTML Codes

File: WoWPro_Leveling/Neutral/SL_Revendreth.lua [hunk 17]
```diff
@@ -1,11 +1,743 @@
...
+A Bottom Feeders|QID|56829|M|65.2,66.2|Z|Revendreth|N|From Rendle.|LVL|57|PRE|57007|
+C Bottom Feeders|QID|56829|M|67.4,62.2|Z|Revendreth|N|Unknown.|
+C Bottom Feeders 1|QID|56829|M|67.4,62.2|Z|Revendreth|QO|1|N|Cudgelface found.|
+C Bottom Feeders 2|QID|56829|M|66.8,57.8|Z|Revendreth|QO|2|N|Lord Chamberlain found.|
+T Bottom Feeders|QID|56829|M|65.2,66.2|Z|Revendreth|N|To Rendle.|
+
+A The Greatest Duelist|QID|57381|M|65.6,64.6|Z|Revendreth|N|From Nadjia the Mistblade.|LVL|57|
+C The Greatest Duelist|QID|57381|M|64.2,65|Z|Revendreth|N|Release Nadjia the Mistblade.|
+C The Greatest Duelist 1|QID|57381|M|64.2,65|Z|Revendreth|QO|1|N|Release Nadjia the Mistblade.|
+C The Greatest Duelist 2|QID|57381|M|66.2,62.2|Z|Revendreth|QO|2|N|<span>Pique Nadjia's Interest.|
```
Comment (Blanckaert) [inline WoWPro_Leveling/Neutral/SL_Revendreth.lua:38]: Extra HTML Codes

**Review by Blanckaert — COMMENTED** [review]
I understand was scrapped version, just noted HTML coding embedded in file.

**Comment by Ludovicus-Maior** [comment]
Mergeing so @Blanckaert can fix

**Pull request merged**